games-strategy/7k-ambition/7k-ambition-3.2.2.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools desktop toolchain-funcs xdg
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.code.sf.net/p/seven-kingdoms-ambition/code"
+	EGIT_BRANCH="next"
+else
+	SRC_URI="https://github.com/Infiltrator/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+DESCRIPTION="Seven Kingdoms: Ambition"
+HOMEPAGE="https://seven-kingdoms-ambition.sourceforge.io/"
+SRC_URI+=" https://sourceforge.net/projects/seven-kingdoms-ambition/files/${PN}.png/download -> ${PN}.png"
+
+LICENSE="GPL-2"
+if [[ ${PV} == *9999* ]]; then
+	KEYWORDS=""
+	SLOT="9999"
+else
+	SLOT="$(ver_cut 1-2)"
+	KEYWORDS="~amd64"
+fi
+
+DEPEND:="dev-libs/boost
+	media-libs/libsdl2[X,video]
+	media-libs/openal
+	net-misc/curl
+	net-libs/enet:1.3"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--prefix="${EPREFIX}/opt/games/${P}" \
+		--datadir="${EPREFIX}/opt/games/${P}/share" \
+		--sysconfdir="${EPREFIX}/opt/games/${P}/etc" \
+		--localedir="${EPREFIX}/opt/games/${P}/share/locale"
```

Comment:
Why using /opt/games instead of the standard paths?

Reply:
Is there no reason, it should follow the standard paths

Reply:
Nope, nevermind.  It doesn't work.

I don't have any way for now on how to separate packages to be installable in the same time.  If you could suggest some changes, I would take the advice.

Reply:
It's a bit more complicated to implement than that and for this case there is no much to gain

Reply:
Okay, we will leave it for the time being.